src/nih_wayfinding/app/scripts/views/routing/overview/overview-controller.js

```diff
@@ -123,6 +129,8 @@
                     resetStyleOnMouseout: true
                 }
             });
+            var walkSpeed = (currentUser.preferences.speed || 2) * 0.44704;
```

Comment:
Would be good to have a note here, or better, put this multiplier in a well-named constant.